starter-code/index.js

```diff
@@ -1,11 +1,53 @@
 class SortedList {
-    constructor() {}
-    add(item) {}
-    get(pos) {}
-    max() {}
-    min() {}
-    average() {}
-    sum() {}
+    constructor() {
+      this.items=[];
+      this.length=0; 
+
+    }
+
+    add(item) {
+      this.items.push(item);
+      this.length+=1; 
+      this.items.sort((a,b)=>{return a-b});
+    }
+
```

Comment:
Ca marche, mais tu peux simplifier encore la syntaxe en écrivant :
`this.items.sort((a,b)=> a-b);`
:)